app/pages/Auction/BidActionPanel/Reserved.js

```diff
@@ -1,26 +1,48 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { shell } from 'electron';
 import {
   AuctionPanel,
 } from '../../../components/AuctionPanel';
-import {I18nContext} from "../../../utils/i18n";
+import { I18nContext } from "../../../utils/i18n";
+
+const INFO_URL = 'https://heytx.substack.com/p/the-happening-event-for-handshake';
```

Comment:
This could come from `locales/en.json` so other languages could link to other blog posts